src/main/java/de/unistuttgart/crosswordbackend/service/ConfigService.java

```diff
@@ -0,0 +1,174 @@
+    return configurationRepository
+      .findById(id)
+      .orElseThrow(() ->
+        new ResponseStatusException(HttpStatus.NOT_FOUND, String.format("There is no configuration with id %s.", id))
+      );
+  }
+
+  /**
+   * Save a configuration
+   *
+   * @param configurationDTO configuration that should be saved
+   * @return the saved configuration as DTO
+   */
+  public ConfigurationDTO saveConfiguration(final ConfigurationDTO configurationDTO) {
+    final Configuration savedConfiguration = configurationRepository.save(
+      configurationMapper.configurationDTOToConfiguration(configurationDTO)
+    );
+    return configurationMapper.configurationToConfigurationDTO(savedConfiguration);
+  }
+
+  /**
+   * Update a configuration
+   *
+   * @throws ResponseStatusException when configuration with the id does not exist
+   * @param id the id of the configuration that should be updated
+   * @param configurationDTO configuration that should be updated
+   * @return the updated configuration as DTO
+   */
+  public ConfigurationDTO updateConfiguration(final UUID id, final ConfigurationDTO configurationDTO) {
+    final Configuration configuration = getConfiguration(id);
+    configuration.setQuestions(questionMapper.questionDTOsToQuestions(configurationDTO.getQuestions()));
+    configuration.setName(configurationDTO.getName());
+    final Configuration updatedConfiguration = configurationRepository.save(configuration);
+    return configurationMapper.configurationToConfigurationDTO(updatedConfiguration);
+  }
+
+  /**
+   * Delete a configuration
+   *
+   * @throws ResponseStatusException when configuration with the id does not exist
+   * @param id the id of the configuration that should be updated
+   * @return the deleted configuration as DTO
+   */
+  public ConfigurationDTO deleteConfiguration(final UUID id) {
+    final Configuration configuration = getConfiguration(id);
+    configurationRepository.delete(configuration);
+    return configurationMapper.configurationToConfigurationDTO(configuration);
+  }
+
+  /**
+   * Add a question to specific configuration
+   *
+   * @throws ResponseStatusException when configuration with the id does not exist
+   * @param id the id of the configuration where a question should be added
+   * @param questionDTO the question that should be added
+   * @return the added question as DTO
+   */
+  public QuestionDTO addQuestionToConfiguration(final UUID id, final QuestionDTO questionDTO) {
+    final Configuration configuration = getConfiguration(id);
+    final Question question = questionRepository.save(questionMapper.questionDTOToQuestion(questionDTO));
+    configuration.addQuestion(question);
+    configurationRepository.save(configuration);
+    return questionMapper.questionToQuestionDTO(question);
+  }
+
+  /**
+   * Delete a question from a specific configuration
+   *
+   * @throws ResponseStatusException when configuration with the id or question with id does not exist
```

Comment:
```suggestion
   * @throws ResponseStatusException (404) when configuration with the id or question with id does not exist
```
I just make it here as demonstration. @delvh and I decided to mark it like this that you know what status code will be thrown. Maybe you will add this here in your code too